### PR TITLE
Added support for s5148f-vm

### PR DESCRIPTION
**Pull request by MaxPolovyi**
* Added support for s5148f-vm
* Using sdi-db-data.sql to simulate sdi

### CODE REVIEW

**Comment by MaxPolovyi** [comment]
@pyadvichuk @akok

File: vendor/dell/s5148f-vm/etc/opx/sdi/sdi-db-data.sql [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+        FROM "Entity_Resource" WHERE Resource_Type = 3;
+
+/* All SFP Media are of the same type */
+DELETE FROM "Media";
+INSERT INTO "Media"
+    SELECT  Entity_Handle, Resource_Handle,
+            1, /* DELL Qualified */
+            1, /* Present */
+            1, /* Speed (10 Gbps) */
+            x'cc37ab', /* EdgeCore OUI TODO */
```
Comment (MaxPolovyi) [inline vendor/dell/s5148f-vm/etc/opx/sdi/sdi-db-data.sql]: Where I can find a correct OUI for the platform?
I'm not sure how important is this value

File: vendor/dell/s5148f-vm/bin/opx_platform_init.sh [hunk 2]
```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Copyright (c) 2018 Dell Inc.
```
Comment (MaxPolovyi) [inline vendor/dell/s5148f-vm/bin/opx_platform_init.sh]: What copyright I should/could use here and in `opx_platform_shutdown.sh`?

File: vendor/dell/s5148f-vm/bin/opx_platform_init.sh [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+#
+# See the Apache Version 2.0 License for specific language governing
+# permissions and limitations under the License.
+
+IS_VM=${IS_VM:-true}
+
+FIRMWARE_VERSION_FILE=/var/log/firmware_versions
+rm -rf ${FIRMWARE_VERSION_FILE}
+
+if [[ ${IS_VM} == true ]]; then
```
Comment (pyadvichuk) [inline vendor/dell/s5148f-vm/bin/opx_platform_init.sh]: please remove this completely. not need to keep both functionality

File: vendor/dell/s5148f-vm/lib/systemd/system/opx-platform-init.service [hunk 4]
```diff
@@ -0,0 +1,15 @@
+[Unit]
```
Comment (pyadvichuk) [inline vendor/dell/s5148f-vm/lib/systemd/system/opx-platform-init.service]: What the difference with hardware based one?

**Comment by MaxPolovyi** [comment]
@andriykokhan , could you please have a look at the PR?

File: vendor/dell/s5148f-vm/lib/systemd/system/opx-platform-init.service (hunk 4 above)
Comment (MaxPolovyi) [reply vendor/dell/s5148f-vm/lib/systemd/system/opx-platform-init.service]: Removed dependency
`After=sys-devices-pci0000:00-0000:00:13.0-i2c ... `

File: vendor/dell/s5148f-vm/bin/opx_platform_init.sh (hunk 3 above)
Comment (MaxPolovyi) [reply vendor/dell/s5148f-vm/bin/opx_platform_init.sh]: Done

**Pull request merged**